internal/controller/services/monitoring/monitoring_controller_support_test.go

```diff
@@ -21,53 +39,72 @@ func TestGetTemplateDataAcceleratorMetrics(t *testing.T) {
 	ctx := t.Context()
 
 	tests := []struct {
-		name                       string
-		managementState            operatorv1.ManagementState
-		metricsConfig              *serviceApi.Metrics
-		expectedAcceleratorMetrics bool
+		name                string
+		monitoringState     operatorv1.ManagementState
+		hasMetricsConfig    bool
+		expectedAccelerator bool
 	}{
 		{
-			name:                       "Accelerator metrics enabled - managed state with metrics config",
-			managementState:            operatorv1.Managed,
-			metricsConfig:              &serviceApi.Metrics{},
-			expectedAcceleratorMetrics: true,
+			name:                "Managed with metrics config",
+			monitoringState:     operatorv1.Managed,
+			hasMetricsConfig:    true,
+			expectedAccelerator: true,
 		},
 		{
-			name:                       "Accelerator metrics disabled - managed state without metrics config",
-			managementState:            operatorv1.Managed,
-			metricsConfig:              nil,
-			expectedAcceleratorMetrics: false,
+			name:                "Managed without metrics config",
+			monitoringState:     operatorv1.Managed,
+			hasMetricsConfig:    false,
+			expectedAccelerator: false,
+		},
+		{
+			name:                "Unmanaged with metrics config",
+			monitoringState:     operatorv1.Unmanaged,
+			hasMetricsConfig:    true,
+			expectedAccelerator: true,
+		},
+		{
+			name:                "Removed with metrics config",
+			monitoringState:     operatorv1.Removed,
+			hasMetricsConfig:    true,
+			expectedAccelerator: true,
 		},
 	}
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			// Create test objects
+			// Create DSCI
 			dsci := &dsciv1.DSCInitialization{
 				ObjectMeta: metav1.ObjectMeta{
 					Name: "test-dsci",
 				},
 				Spec: dsciv1.DSCInitializationSpec{
 					Monitoring: serviceApi.DSCIMonitoring{
 						ManagementSpec: common.ManagementSpec{
-							ManagementState: tt.managementState,
+							ManagementState: tt.monitoringState,
 						},
 					},
 				},
 			}
 
+			// Create Monitoring object
 			monitoring := &serviceApi.Monitoring{
 				ObjectMeta: metav1.ObjectMeta{
 					Name: "test-monitoring",
```

Comment:
I think this should be named default-monitoring, there is validation in place that should enforce that as the only accepted name.